src/lib/pages/home-page-data.ts

```diff
@@ -0,0 +1,41 @@
+import {sanityClient} from 'utils/sanity-client'
+import groq from 'groq'
+
+export async function loadHomePageData() {
+  const slugs = {
```

Comment:
these are named slugs to use as variables in the composed query below you'll see it referenced as `$jumbotron`

this should mean we can swap it out with a different slug and it'll "just work". You can also use objects and reference them like `$object.slug` if you want to get more tricky.